dev-lang/jimtcl/files/jimtcl-0.78-jimsh0-cflags.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/autosetup/autosetup-find-tclsh b/autosetup/autosetup-find-tclsh
+index a0f1ae6..1145845 100755
+--- a/autosetup/autosetup-find-tclsh
++++ b/autosetup/autosetup-find-tclsh
+@@ -7,7 +7,7 @@ d=`dirname "$0"`
+ PATH="$PATH:$d"; export PATH
+ echo 1>&2 "Bootstrap jimsh0"
+ for cc in ${CC_FOR_BUILD:-cc} gcc; do
+-	{ $cc -o "$d/jimsh0" "$d/jimsh0.c"; } 2>/dev/null || continue
++	{ $cc -o "$d/jimsh0" "$d/jimsh0.c" $CFLAGS; } 2>/dev/null || continue
```

Comment:
Also LDFLAGS and CPPFLAGS, please, and preferably in standard make order.

Reply:
Should we really pass `CPPFLAGS` to C compiler? Could it contain incompatible options?

Reply:
Yes, these are preprocessor flags.